### PR TITLE
Initial Use Cases

### PR DESCRIPTION
**Pull request by jsalowey**
Closes Issue #4

### CODE REVIEW

File: draft-salowey-wimse-arch.md [hunk 1]
```diff
@@ -55,10 +56,44 @@ TODO Introduction
 
 # Use Cases
 
+1. Basic Service Authentication
+
+One of the most basic use cases for workload identity is for authenticating one workload to another such as in the case where one microservice is making a request of another microservice within a larger application. Even in this simple case the identity of the workload is often a composite of many attributes such as:
```
Comment (yaroslavros) [inline draft-salowey-wimse-arch.md:61]: Does not have to be "micro"service. I would suggest generalising this to any service.

File: draft-salowey-wimse-arch.md [hunk 2]
```diff
@@ -55,10 +56,44 @@ TODO Introduction
...
+* Hardware Attestation
+
+These attributes are used for various purposes:
+
+* ensuring the request is made to the correct service or service instance
+* authorizing access to APIs and resources
+* providing an audit train of requests within a system
+* providing context for other decisions made within a service
+
+There are several methods defined to perform this authentication.  SOme of the most common include:
```
Comment (yaroslavros) [inline draft-salowey-wimse-arch.md:78]: SOme -> Some

**Review by yaroslavros — COMMENTED** [review]
Great start!

File: draft-salowey-wimse-arch.md (hunk 1 above)
Comment (jsalowey) [reply draft-salowey-wimse-arch.md:61]: Agree

**Pull request merged**